go/vt/mysqlctl/mysqld.go

```diff
@@ -165,6 +165,7 @@ func NewMysqld(dbcfgs *dbconfigs.DBConfigs) *Mysqld {
 				os.Getenv("VTROOT"),
 				vtenvMysqlRoot,
 				os.Getenv("MYSQL_FLAVOR"))
+			log.Errorf(message)
```

Comment:
errant Log message or do we want to commit this?

Reply:
Yes we need it. I saw during startup if we don't able to find MySQL then error log does not have any information.